job-tutorial/Dockerfiles/cuda9.0-cudnn7/Dockerfile.build.base

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.'
+
+
+# tag: pai.build.base:hadoop2.7.2-cuda9.0-cudnn7-devel-ubuntu16.04
+#
+# Base image to build for the system.
+# Other images depend on it, so build it like:
+#
+# docker build -f Dockerfile.build.base -t pai.build.base:hadoop2.7.2-cuda9.0-cudnn7-devel-ubuntu16.04 .
+
+
+# Tag: nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+# Label: com.nvidia.cuda.version: 9.0.176
+# Label: com.nvidia.cudnn.version: 7.1.2.21
+# Label: com.nvidia.volumes.needed: nvidia_driver
+# Label: maintainer: NVIDIA CORPORATION <cudatools@nvidia.com>
+# Ubuntu 16.04
+FROM nvidia/cuda@sha256:40db1c98b66e133f54197ba1a66312b9c29842635c8cba5ae66fb56ded695b7c
```

Comment:
Is cuda9.0-cudnn7 can work with our current version of NVidia driver?

Reply:
yes, here're [compatibility charts](https://stackoverflow.com/questions/30820513/what-is-version-of-cuda-for-nvidia-304-125/30820690#30820690).